config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -33,9 +33,9 @@ spec:
     official support.
 
     ### Pre-requisites
-    * A Red Hat OpenShift Container Platform for Windows Containers subscription
-    * OCP 4.10 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
-    * [WMCO prerequisites](https://docs.openshift.com/container-platform/4.10/windows_containers/understanding-windows-container-workloads.html#wmco-prerequisites__understanding-windows-container-workloads)
+    * A Red Hat OpenShift Container Platform for Windows Containers [subscription](https://access.redhat.com/support/policy/updates/openshift#windows)
+    * OCP 4.11 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
```

Comment:
There is still a ref to 4.6 in line [#138](https://github.com/openshift/windows-machine-config-operator/pull/1088/files#diff-2aebce80f6b12b915e64655039782d17d315ad5b3ed43c37517cc6556022792cR138). BTW, the troubleshooting page is missing in 4.10 docs.

Reply:
Nice catch.  And 4.6 on line #48.

Reply:
> BTW, the troubleshooting page is missing in 4.10 docs

cc: @mburke5678 

https://docs.openshift.com/container-platform/4.10/support/troubleshooting/troubleshooting-windows-container-workload-issues.html doesn't seem to exist but works fine for 4.9 and prior.

Reply:
Thanks @mburke5678 for fixing the troubleshooting page

- https://github.com/openshift/openshift-docs/pull/46334